test/tests/ver-1g/equal_conc.i

```diff
@@ -0,0 +1,14 @@
+[ICs]
+    ca_IC = '${fparse ${units 1 muPa -> Pa} * ${Na} / ( ${R} * ${T} ) }'
+    cb_IC = '${fparse ${units 1 muPa -> Pa} * ${Na} / ( ${R} * ${T} ) }'
+    [ca_same_conc_IC]
+        type = ConstantIC
+        variable = c_a
+        value = '${units ${ca_IC} at/m^3 -> at/mum^3 }'
```

Comment:
Why not do the conversion on line 2?

Reply:
Combining it all on one line made it hard to read. The current approach might actually be better.